tests/__tests__/mixins/labels.js

```diff
@@ -1,25 +1,44 @@
 import { ATOM_GROUP_NAME } from "../../../src/enums";
 import { getWaveInstance } from "../../enums";
+import * as THREE from "three";
 
 describe("Atom labels", () => {
     let wave, atoms, labels;
 
     beforeEach(() => {
         wave = getWaveInstance();
         const atomGroup = wave.scene.getObjectByName(ATOM_GROUP_NAME);
+        labels = wave.structureGroup.children.filter((child) => child instanceof THREE.Points);
         atoms = atomGroup.children.filter((object) => object.type === "Mesh");
-        labels = atomGroup.children.filter((object) => object.type === "Sprite");
     });
 
     test("Labels are created for every atom", async () => {
```

Comment:
Per https://github.com/Exabyte-io/wave.js/pull/100/files#r940467021 below - this should be called: "Labels are created for every atom and placed at their centers" probably